app/konnect/runtime-manager/plugins.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Plugins
+content_type: explanation
+---
+
+You can extend {{site.konnect_short_name}} by using plugins. Kong provides a set of standard Lua plugins that get bundled with {{site.konnect_short_name}}. The set of plugins you have access to depends on your installation.
```

Comment:
Maybe this is a good opportunity to link to some of the docs on desigining plugins? Or the plugin design template 

https://github.com/Kong/kong-plugin


I don't know if this works w/ konnect but it feels rather empty to say that things can be extended but not show people how or why.

Reply:
Could also remove that section